tools/bpf2c/bpf_code_generator.h

```diff
@@ -13,6 +13,8 @@
 #include <string>
 #include <vector>
 
+#define EBPF_HASH_ALGORITHM "SHA256"
```

Comment:
Add a comment saying the value must be one of https://learn.microsoft.com/en-us/windows/win32/seccng/cng-algorithm-identifiers

Reply:
Fixed.